services/github/github-helpers.js

```diff
@@ -24,6 +25,14 @@ function errorMessagesFor(notFoundMessage = 'repo not found') {
   }
 }
 
+function graphqlErrorHandler(errors) {
+  if (errors[0].type === 'NOT_FOUND') {
+    throw new NotFound({ prettyMessage: 'repo not found' })
+  } else {
```

Comment:
In time, it might be useful to add other cases in here, but I don't know what they are yet.